### ISSUE TITLE
[Core] Screen reader support

### ISSUE THREAD

**Issue by Zitrone44** [thm-mni-ii/feedbacksystem#1747]
**Is your feature request related to a problem? Please describe.**
the fbs should be accessible with a screenreader.

**Describe the solution you'd like**
* [x] Courses can't be selected with TAB.
* [x] Skip main menu
* [x] Skip page menu
* [x] Can't use tab to escape the SQL-Playground (Shift+Enter can now be used to focus on the enter button)
